src/PointwiseFunctions/GeneralRelativity/WeylElectric.cpp

```diff
@@ -0,0 +1,63 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/GeneralRelativity/WeylElectric.hpp"
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+#include "Utilities/MakeWithValue.hpp"
+
+namespace gr {
+template <size_t SpatialDim, typename Frame, typename DataType>
+tnsr::ii<DataType, SpatialDim, Frame> weyl_electric(
+    const tnsr::ii<DataType, SpatialDim, Frame>& spatial_ricci,
+    const tnsr::ii<DataType, SpatialDim, Frame>& extrinsic_curvature,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept {
+  auto weyl =
+      make_with_value<tnsr::ii<DataType, SpatialDim, Frame>>(spatial_ricci, 0.);
+  constexpr auto dimensionality = index_dim<0>(weyl);
+  for (size_t i = 0; i < dimensionality; ++i) {
+    for (size_t j = i; j < dimensionality; ++j) {
+      weyl.get(i, j) += spatial_ricci.get(i, j);
+
+      for (size_t k = 0; k < dimensionality; ++k) {
+        for (size_t l = 0; l < dimensionality; ++l) {
+         weyl.get(i, j) += extrinsic_curvature.get(k, l) *
+                           inverse_spatial_metric.get(k, l) *
+                           extrinsic_curvature.get(i, j);
+
+         for (size_t m = 0; m < dimensionality; ++m) {
+             weyl.get(i, j) -= extrinsic_curvature.get(i, l) *
+                              inverse_spatial_metric.get(m, l) *
+                              extrinsic_curvature.get(m, j);
+        }
```

Comment:
I think a see a problem here. There is no "k" in the -= term, yet you're summing over k. This will give you 4 times too much for this term, because each time through the k look, this -= will happen. Probably you can rewrite the equation as 

E_{ij} = R_{ij} + g^{kl}K_{kl}K_{ij} - g^{kl}K_{il}K_{kj}

and then get rid of the m loop